docs/components/policy/resource_mappings.md

```
@@ -6,11 +6,63 @@ A Resource Mapping contains:
+
+A Resource Mapping Group contains:
+
+1. A **name** identifying the group.
+2. A **namespace** that the group belongs to.
+3. A **Fully Qualified Name (FQN)** derived from the namespace and group name.
+
+### FQN Format
+
+Each Resource Mapping Group has a unique FQN following this pattern:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)